integration/entropy/entropy_time.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source /etc/os-release || source /usr/lib/os-release
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	clean_env
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "measured time for /dev/random" {
+	output_file=$(mktemp)
+	block_size="4b"
+	expected_time="20"
+	copy_number="100000"
+	image="busybox"
+	cmd="time dd if=/dev/random of=/dev/null bs=$block_size count=$copy_number"
```

Comment:
do we really need `time` ? I see that later you are doing `.. | grep seconds`, which comes from the output of `dd`

Reply:
ah, if `dd` gives the time, then sure we can use that. I my home run local example I was thinking of the seconds from the time. heh.